migration/migration/012-tracing.sql

```diff
@@ -58,10 +58,10 @@ CREATE TABLE _ps_trace.tag
     tag_type ps_trace.tag_type NOT NULL,
     key_id bigint NOT NULL,
     key ps_trace.tag_k NOT NULL REFERENCES _ps_trace.tag_key (key) ON DELETE CASCADE,
-    value ps_trace.tag_v NOT NULL,
-    UNIQUE (key, value) INCLUDE (id, key_id)
+    value ps_trace.tag_v NOT NULL
```

Comment:
As I understand it, we cannot modify this file `012-tracing.sql`, rather we have to add a new migration file which would assume the original version of the table exists and issue ALTERs etc. to make this change.